src/ast/mod.rs

```diff
@@ -4050,6 +4051,13 @@ pub enum Statement {
         arguments: Vec<Expr>,
         options: Vec<RaisErrorOption>,
     },
+    /// Return (Mssql)
+    ///
+    /// for Functions:
+    /// RETURN scalar_expression
+    ///
+    /// See: https://learn.microsoft.com/en-us/sql/t-sql/statements/create-function-transact-sql
```

Comment:
```suggestion
    /// Return
    ///
    /// for Functions:
    /// RETURN scalar_expression
    ///
    /// [MsSql]: https://learn.microsoft.com/en-us/sql/t-sql/statements/create-function-transact-sql
```
since the statement is supported by other dialects